apps/web/views/pets/update.rb

```diff
@@ -0,0 +1,7 @@
+module Web::Views::Pets
```

Comment:
I didn't get yet the idea of  `views`, `templates` and `controllers`.

I will take I look on the docs :grin: